kofta/public/locales/de/translation.json

```diff
@@ -1,156 +1,165 @@
 		},
 		"modals": {
-			"createRoomModal": { "public": "public", "private": "private" },
+			"createRoomModal": {
+				"public": "öffentlich",
+				"private": "privat"
+			},
 			"invitedToJoinRoomModal": {
-				"newRoomCreated": "New Room Created",
-				"roomInviteFrom": "Room Invite from",
-				"justStarted": "They just started",
-				"likeToJoin": ", would you like to join?",
-				"inviteReceived": "you've been invited to"
+				"newRoomCreated": "Neuer Raum erstellt",
+				"roomInviteFrom": "Neue Einladung von",
+				"justStarted": "Sie starten gerade",
+				"likeToJoin": ", willst du beitreten?",
+				"inviteReceived": "Du wurdest eingeladen zu"
 			},
 			"editProfileModal": {
-				"usernameTaken": "username taken",
-				"avatarUrlError": "Invalid image",
-				"avatarUrlLabel": "Github/Twitter avatar url",
-				"displayNameError": "length 2 to 50 characters",
-				"displayNameLabel": "Display Name",
-				"usernameError": "length 4 to 15 characters and only alphanumeric/underscore",
-				"usernameLabel": "Username",
-				"bioError": "max length of 160 characters",
+				"usernameTaken": "Benutzername bereits vergeben",
+				"avatarUrlError": "Ungültiges Bild",
+				"avatarUrlLabel": "Github/Twitter Avatar URL",
+				"displayNameError": "länge 2 bis 50 Buchstaben",
+				"displayNameLabel": "Anzeigename",
+				"usernameError": "länge 4 bis 15 alphanumerische Zeichen und Unterstriche",
+				"usernameLabel": "Benutzername",
+				"bioError": "maximal 160 Buchstaben",
 				"bioLabel": "Bio"
 			},
 			"profileModal": {
-				"blockUserConfirm": "Are you sure you want to block this user from joining any room you ever create?",
-				"blockUser": "block user",
-				"makeMod": "make mod",
-				"unmod": "unmod",
-				"addAsSpeaker": "add as speaker",
-				"moveToListener": "move to listener",
-				"banFromChat": "ban from chat",
-				"banFromRoom": "ban from room",
-				"goBackToListener": "go back to listener",
-				"deleteMessage": "delete this message"
+				"blockUserConfirm": "Bist du sicher, dass du diesen Benutzer aus allen deinen zukünftigen Räumen bannen willst?",
+				"blockUser": "benuzter bannen",
+				"makeMod": "zum moderator befördern",
+				"unmod": "moderatorstatus aufheben",
+				"addAsSpeaker": "zu sprecher befördern",
+				"moveToListener": "zu zuhörer herabstufen",
+				"banFromChat": "aus chat bannen",
+				"banFromRoom": "aus raum bannen",
+				"goBackToListener": "zurück zu zuhörer",
+				"deleteMessage": "nachricht löschen"
 			},
 			"roomSettingsModal": {
-				"requirePermission": "require permission to speak",
-				"makePublic": "make room public",
-				"makePrivate": "make room private"
+				"requirePermission": "berechtigung zum sprechen verlangen",
+				"makePublic": "raum öffentlich machen",
+				"makePrivate": "raum privat machen"
 			}
 		},
 		"userVolumeSlider": {
-			"noAudioMessage": "no audio consumer for some reason"
+			"noAudioMessage": "aus unbekannten gründen kein audio consumer"
```

Comment:
consumer = konsument (?)

Reply:
Ich würde hier consumer lassen, da es ein eher ein technischer Begriff aus mediasoup ist.
English: I'd leave consumer here, since it's a technical term mediasoup uses.